web/vtadmin/src/components/routes/Workflows.tsx

```diff
@@ -70,6 +72,12 @@ export const Workflows = () => {
                 <tr key={idx}>
                     <DataCell>
                         <div className="font-bold">{href ? <Link to={href}>{row.name}</Link> : row.name}</div>
+                        <div className="text-secondary text-success-200">
+                            {row.workflowType}
```

Comment:
```suggestion
                            {row.workflowType || 'N/A'}
```

Reply:
^ Just adding a nil check for `row.workflowType` - feel free to change from 'N/A' if something else is appropriate.